charts/fleet-agent/ci/default-values.yaml

```diff
@@ -0,0 +1,2 @@
+apiServerURL: "https://localhost"
```

Comment:
Would this file, and other files under `charts/fleet{-agent,}/ci` end up being packaged in our charts?
Should we add `.helmignore` files to those charts to prevent that?

Apart from that, I'm sligthly concerned about the maintenance overhead of those values files when we add new features involving such values: it looks likely to me that we may forget to update at least some of them to reflect the latest available values.

Reply:
Thanks, added those `.helmignore` files. The actual values in those files don't matter much. I think chart-testing just checks that the result is valid YAML.